apps/financial-aid/web-veita/src/components/ModalTypes/AcceptModal.tsx

```diff
@@ -81,7 +85,7 @@ const AcceptModal = ({
 
   const [state, setState] = useState<calculationsState>({
     amount: aidAmount,
-    childrenAidAmount: 0,
+    childrenAidAmount: showChildren ? undefined : 0,
```

Comment:
I would personally always initialise this to undefined and then had the null coalescing logic where it is used since you will always have to do that anyways.